public/app/features/alerting/unified/hooks/AI/useAIRuleFeedback.test.ts

```diff
@@ -0,0 +1,133 @@
+import { act, renderHook } from '@testing-library/react';
+
+import { config } from '@grafana/runtime';
+
+import { testWithFeatureToggles } from '../../test/test-utils';
+
+import { useAIRuleFeedback } from './useAIRuleFeedback';
+
+const mockUseURLSearchParams = jest.fn();
+jest.mock('../useURLSearchParams', () => ({
+  useURLSearchParams: () => mockUseURLSearchParams(),
+}));
+
+describe('useAIRuleFeedback', () => {
+  const mockTracker = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('with feature toggle enabled', () => {
+    testWithFeatureToggles(['alertingAIGenAlertRules']);
+
+    it('should return isFromAI as true when fromAI param is true', () => {
+      mockUseURLSearchParams.mockReturnValue([new URLSearchParams('fromAI=true'), jest.fn()]);
+
+      const { result } = renderHook(() =>
+        useAIRuleFeedback(mockTracker, config.featureToggles.alertingAIGenAlertRules === true)
```

Comment:
Why we pass `config.featureToggles.alertingAIGenAlertRules === true` in almost all tests?

Reply:
ops, there is no need to pass this...let's pass directly a boolean 💯